### ISSUE TITLE
Contrast issue on link hover

### ISSUE THREAD

**Issue by transcental** [rayhanadev/personal-website-v4#1]
![Screenshot 2023-02-02 104228](https://user-images.githubusercontent.com/83948303/216314986-863cade3-bff1-4aab-8c77-7edf4d08eae3.png)
When hovering over a link, the link can't easily be read.

**Comment by rayhanadev** [comment]
Right! Clement mentioned that. Let me fix it thanks for bringing it to my attention again :).
